src/StatusQ/Components/StatusListItem.qml

```diff
@@ -23,7 +23,6 @@ Rectangle {
     property string titleTextIcon: ""
     property real leftPadding: 16
     property real rightPadding: 16
-    property bool enabled: true
```

Comment:
Self note: this was a pain to discover :)

Reply:
This is a breaking change and should be denoted as such.
Also gotta make sure this will be taken into account in desktop

Reply:
It's not a breaking change; it was just wrongly shadowing an already existing property (https://doc.qt.io/qt-5/qml-qtquick-item.html#enabled-prop), this is just fixing it

Reply:
Ah cool!